airbyte-integrations/connectors/source-zendesk-support/metadata.yaml

```
@@ -7,7 +7,7 @@ data:
   connectorType: source
   maxSecondsBetweenMessages: 10800
   definitionId: 79c1aa37-dae3-42ae-b333-d1c105477715
-  dockerImageTag: 0.3.1
+  dockerImageTag: 0.4.0
```

If this is backwards compatible, it can just be a minor version change

```suggestion
  dockerImageTag: 0.3.2
```